projects/core/src/cms/config/cms-config.ts

```diff
@@ -35,6 +35,7 @@ export interface CmsComponentMapping {
   providers?: StaticProvider[];
   childRoutes?: Routes;
   disableSSR?: boolean;
+  namespacesI18N?: string[];
```

Comment:
I know it is mater of personal preference, but did you thought about naming it 'i18nNamespaces'?